Resources/config/core.xml

```diff
@@ -66,7 +66,7 @@
         </service>
         <!-- twig -->
         <service id="sonata.admin.twig.global" class="Sonata\AdminBundle\Twig\GlobalVariables">
-            <argument type="service" id="service_container"/>
+            <argument type="service" id="sonata.admin.pool"/>
```

Comment:
Again,
![workflow](https://imgs.xkcd.com/comics/workflow.png)


If you set the troll level high enough, every PR is a BC-break. The real question is "how high do we set the threshold for this BC-break to be considered major?". And I think this one is under the threshold (the percentage of users doing that is estimated to very low).